# The header contains sequence number and timestamp,
# marker_position are the found marker positions in the camera image
Header header
int32 cameraID
Vector2[] marker_position
